docs/source/standard/bid.rst

```diff
@@ -10,12 +8,26 @@ Bid
 Schema
 ------
 
+:status:
+    string, default='active'
+
+    Possible values are:
+
+    * `draft`
+    * `active`
+
 :tenderers:
-    List of :ref:`Organization` objects
+    List of :ref:`Organization` objects, required
+
+:documents:
+    List of :ref:`Document` objects
+
+:qualified:
```

Comment:
Changing the location of the same items is a bad practice. Please avoid doing this.